examples/3d/edit_gltf_scene.rs

```diff
@@ -0,0 +1,100 @@
+//! Edit a scene from a glTF file, before spawning the scene in the world
+
+use bevy::{prelude::*, reflect::TypeUuid};
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_system(setup_scenes)
+        .run();
+}
+
+pub const CAKE_WITH_LIGHT_HANDLE: HandleUntyped = HandleUntyped::weak_from_u64(Scene::TYPE_UUID, 1);
```

Comment:
I would add a comment explaining how these constants were chosen.

Reply:
would you believe me if I said "randomly selected value"? 😄

Reply:
*grin* Of course ;) Good to explain to the users though.